library/psa_crypto.c

```diff
@@ -443,7 +443,7 @@ mbedtls_ecp_group_id mbedtls_ecc_group_of_psa( psa_ecc_family_t curve,
 
 static psa_status_t prepare_raw_data_slot( psa_key_type_t type,
                                            size_t bits,
-                                           struct raw_data *raw )
+                                           struct key_data *raw_key )
```

Comment:
NOT a comment on this line but on the commit message.
It seems to me that the commit message is only weakly connected to what is done in this commit. Please rather explain why the renaming/redefinition is needed/beneficial.

Reply:
Rewrote history and fixed.